components/ActivityList.js

```diff
@@ -0,0 +1,20 @@
+import styled from "styled-components";
+import { activities } from "@/libs/activities";
+import ActivityCard from "@/components/ActivityCard";
+
+export default function ActivityList() {
+  return (
+    <StyledMain>
+      {activities.map((activity) => (
+        <ActivityCard key={activity.id} {...activity} />
+      ))}
+    </StyledMain>
```

Comment:
you are mapping here, therefore you should wrap the mapping in a `ul` and the mapped elements in an `li`. Don't forget to move the key to the li.